test/hello_world.2asm.js

```diff
@@ -13,8 +13,8 @@ function asmFunc(global, env, buffer) {
  var Math_abs = global.Math.abs;
  var Math_clz32 = global.Math.clz32;
  function add(x, y) {
-  x = x | 0;
-  y = y | 0;
+  x = (x | 0)
```

Comment:
this looks worse than before - the parens are not needed, and the `;` is not strictly needed but is safer to emit. it's actually possible some asm.js compilers depend on the lack of parens and the existence of the ';', I'm not sure - the spec can be read as assuming the very canonical form `x = x | 0;` (up to whitespace)